src/main/webapp/WEB-INF/tags/instructor/comments/commentsForStudentsPanel.tag

```diff
@@ -41,39 +47,27 @@
                                             onclick="return deleteComment('${commentIdx}');"
                                             data-toggle="tooltip"
                                             data-placement="top"
-                                            title=""
-                                            data-original-title="<%= Const.Tooltips.COMMENT_DELETE %>"
+                                            title="<%= Const.Tooltips.COMMENT_DELETE %>"
```

Comment:
Good catch, we need to fix this for other files as well such as

![screen shot 2015-07-09 at 5 35 32 pm](https://cloud.githubusercontent.com/assets/6291947/8592321/f50e0ffc-2660-11e5-9a82-82615eab3299.png)

whoever copy pasted stuff just tried to do what bootstrap would have done manually